tuf/ngclient/config.py

```diff
@@ -15,3 +15,6 @@ class UpdaterConfig:
     timestamp_max_length: int = 16384  # bytes
     snapshot_max_length: int = 2000000  # bytes
     targets_max_length: int = 5000000  # bytes
+    # By default, consistent_snapshot is set to true to provide uniquely
+    # identifiable metadata and targets file names for repositories.
+    consistent_snapshot: bool = True
```

Comment:
`consistent_snapshot` is set by the repository. The client can/should choose whether targets are hash prefixed or not. See comment below.